### PR TITLE
Fix image size before loading it

### PR DESCRIPTION
**Pull request by vitsalis**

### CODE REVIEW

**Comment by vitsalis** [comment]
Fixes #314

**Comment by dionyziz** [comment]
merge master

File: static/script/user/view.js [hunk 1]
```diff
@@ -23,11 +25,13 @@ var UserView = {
 
         if ( imgWidth > imgHeight ) {
             $image.height( UserView.IMAGE_HEIGHT );
+            $image.width( Math.floor( UserView.IMAGE_HEIGHT * imgWidth / imgHeight ) );
```
Comment (dionyziz) [inline static/script/user/view.js:28]: This seems to work, but why is it needed? The browser should calculate the appropriate width, right?

File: static/script/user/view.js [hunk 2]
```diff
@@ -3,12 +3,14 @@ var UserView = {
     IMAGE_HEIGHT: 168,
     IMAGE_WIDTH: 168,
     showUploadedImage: function( source ) {
-        $( ".avatar img" ).remove();
         $image = $( '<img src="' + source + '" alt="Profile Picture" />' );
         $image.load( function() {
+            $image.width( this.width );
+            $image.height( this.height );
```
Comment (dionyziz) [inline static/script/user/view.js:9]: I'm confused as to why you're setting these. Are you setting the width to what it already is?

**Pull request merged**